src/automation/auto_inventory.py

```diff
@@ -0,0 +1,108 @@
+'''
```

Comment:
since this is a main script, add a shebang? e.g., #!/usr/bin/env python

Reply:
as this entire repo, including this script, requires running the obs_inv_utils_{platform}.sh files before hand (I will add that to the documentation as I do it automatically now), a shebang would not be necessary nor I believe the appropriate usage in this case, especially since we need to use the python from the stack.